docs/features/configuration.md

```diff
@@ -24,10 +25,14 @@ Before running any containers Testcontainers will perform a set of startup check
 It takes a couple of seconds, but if you want to speed up your tests, you can disable the checks once you have everything configured. Add `checks.disable=true` to your `$HOME/.testcontainers.properties` to completely disable them.
 
 ## Customizing images
+
+Testcontainers uses public Docker images to perform different actions like startup checks, VNC recording and others. 
+Some companies disallow the usage of Docker Hub, but you can override `*.image` properties with your own images from your private registry to workaround that.
+
 > **tinyimage.container.image = alpine:3.5**  
 > Used by Testcontainers' core
 
-> **vncrecorder.container.image = richnorth/vnc-recorder:latest**  
+> **vncrecorder.container.image = quay.io/testcontainers/vnc-recorder:1.1.0**  
 > Used by VNC recorder in Testcontainers' Seleniun integration
 
 > **ambassador.container.image = richnorth/ambassador:latest**  
```

Comment:
not used anymore :)

Reply:
Good point. It's present in code still, so I'll raise another ticket to refactor that out entirely.

Reply:
Raised #1165.